src/App.js

```diff
@@ -1,23 +1,93 @@
-import logo from './logo.svg';
+//import logo from './logo.svg';
+import { useState } from 'react';
 import './App.css';
 
+import IsOpen from './OpenSign';
+import Parade from './AnimalParade';
+
+import TigerKing from '../src/HeyAllYou.mp3';
+import CaroleBasking from '../src/ImNever.mp3';
+
 function App() {
+  //track states for stop lights, animal sizes, and animal parade
+  const audio = new Audio(TigerKing);
+  const audioTwo = new Audio(CaroleBasking);
+
+  const [isOpen, setIsOpen] = useState(true);
+
+  function playTigerKing() {
+    audio.play();
+  }
+  function playCaroleBaskin() {
+    audioTwo.play();
+  }
+
+  function handleOpenZoo() {
+    setIsOpen(true);
+    playTigerKing();
+  }
+  function handleCloseZoo() {
+    setIsOpen(false);
+    playCaroleBaskin();
+  }
+
+  const [tigerKingSize, setTigerKingSize] = useState(10);
+  const [tigerQueenSize, setTigerQueenSize] = useState(10);
+
+  const [traffic, setTraffic] = useState(['cheetah', 'tiger']);
```

Comment:
since we're dealing with animals and not vehicles, consider our naming conventions and maybe call these `animals` and not `traffic`